data/redirects.yml

```diff
@@ -113,6 +113,10 @@
   # used in warnings printed by the Docker Engine, and in the installation script
   # at "get.docker.com"
   - /go/attack-surface/
+"/engine/security/protect-access/":
+  # Details about how to protect the Docker daemon socket.
+  # This redirect is used in warnings printed by the Docker Engine.
+  - /go/protect-daemon/
```

Comment:
Trying to think if it needs to have something in the URL that makes it clear it's about the API. Naming is hard though!! @vvoland @neersighted in case you have good ideas / suggestions.

Reply:
* socket-security
* api-security
* security-model
* privileged-socket
* rootful-daemonful

Reply:
`api-security` may work. some others;

- `api-access`
- `api-protection`